crowbar_framework/app/models/node_object.rb

```diff
@@ -1123,6 +1123,16 @@ def wait_for_script_to_finish(script, seconds, args = [])
     end
   end
 
+  # Removes the -ok/-failed files that might have been created by a script
+  # running via "wait_for_script_to_finish"
+  def delete_script_exit_files(script)
+    base = "/var/lib/crowbar/upgrade/" + File.basename(script, ".sh")
+    ok_file = base + "-ok"
+    failed_file = base + "-failed"
```

Comment:
It would probably make sense to have a method that would return `[ ok_file, failed_file ]` for given script. We're using this mechanism already on 3 places in this model...